README.md

```diff
@@ -25,16 +25,19 @@ Hackable [implementation](lit_gpt/model.py) of state-of-the-art open-source larg
-| Together [RedPajama-INCITE](tutorials/download_redpajama_incite.md) | [Together 2023](https://together.ai/blog/redpajama-models-v1)                                    |
-| EleutherAI [Pythia](tutorials/download_pythia.md)                   | [Biderman et al. 2023](https://arxiv.org/abs/2304.01373)                                         |
-| StabilityAI [StableLM](tutorials/download_stablelm.md)              | [Stability AI 2023](https://github.com/Stability-AI/StableLM)                                    |
+| Model and usage                                                                | Reference                                                                                        |
+|--------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| Meta AI [Llama 2](tutorials/download_llama_2.md)                               | [Touvron et al. 2023](https://arxiv.org/abs/2307.09288)                                          |
+| LMSYS [LongChat](tutorials/download_longchat.md)                               | [LongChat Team 2023](https://lmsys.org/blog/2023-06-29-longchat/)                                |
+| Stability AI [FreeWilly2](tutorials/download_freewilly_2.md)                   | [Stability AI 2023](https://stability.ai/blog/stable-beluga-large-instruction-fine-tuned-models) |
+| TII UAE [Falcon](tutorials/download_falcon.md)                                 | [TII 2023](https://falconllm.tii.ae)                                                             |
+| OpenLM Research [OpenLLaMA](tutorials/download_openllama.md)                   | [Geng & Liu 2023](https://github.com/openlm-research/open_llama)                                 |
+| LMSYS [Vicuna](tutorials/download_vicuna.md)                                   | [Li et al. 2023](https://lmsys.org/blog/2023-06-29-longchat)                                     |
+| Together [RedPajama-INCITE](tutorials/download_redpajama_incite.md)            | [Together 2023](https://together.ai/blog/redpajama-models-v1)                                    |
+| EleutherAI [Pythia](tutorials/download_pythia.md)                              | [Biderman et al. 2023](https://arxiv.org/abs/2304.01373)                                         |
+| StabilityAI [StableLM](tutorials/download_stablelm.md)                         | [Stability AI 2023](https://github.com/Stability-AI/StableLM)                                    |
+| Platypus                                                                       | [Lee, Hunter, and Ruiz 2023](https://arxiv.org/abs/2308.07317)                                   |
+| NousResearch Nous-Hermes                                                       | [Model Card](https://huggingface.co/NousResearch/Nous-Hermes-13b)                                |
```

Comment:
We support several NousResearch models. Should we link to the org page instead?
```suggestion
| NousResearch Nous-Hermes                                                       | [Model Card](https://huggingface.co/NousResearch)                                |
```

Reply:
Yes, agreed